pepper-apis/src/main/resources/db-changes/schema/DDP-4124-create-per-activity-validation-tables.xml

```diff
@@ -5,17 +5,18 @@
                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
 
     <changeSet author="sretsia" id="20191024-create-activity-validation-table">
+        <validCheckSum>8:efe74766758312dd4fdbde8846be8416</validCheckSum>
```

Comment:
There's an issue where if you ran liquibase on a fresh new local db running utf8mb4, liquibase will fail because existing liquibase scripts will exceed mysql size limits. This is not the most elegant solution, but idea here is to change this file to be within size limits and tell liquibase the new checksum of file.

Reply:
validCheckSum should work